phinx.yml

```diff
@@ -1,5 +1,7 @@
 paths:
-    migrations: %%PHINX_CONFIG_DIR%%/db/migrations
+    migrations:
```

Comment:
As this is the like file that is supplied, I don't think it should have any additional paths.